README.adoc

```diff
@@ -241,18 +251,18 @@ The helper function invokes the `system` service by calling the [hotspot=61 file
 The `inventory` service might also connect with a host other than the default `localhost` host. Then, you cannot configure a base URL that is not yet known.
```

Comment:
Change
The `inventory` service might also connect with a host other than the default `localhost` host. Then, you cannot configure
TO
The `inventory` service **can** connect with a host other than the default `localhost` host. **You cannot then**

Reply:
Adjusted